README.md

```diff
@@ -23,18 +23,19 @@ step certificate create identity.linkerd.cluster.local issuer.crt issuer.key --p
 - Finally construct your user secrets file by filling this template and saving as `my-linkerd-certificates.yaml`:
 
 ```yaml
-identity:
-  issuer:
-    tls:
-      crtPEM: |
-        <contents of the issuer.crt file>
-      keyPEM: |
-        <contents of the issuer.key file>
-identityTrustAnchorsPEM: |
-  <contents of the ca.crt file>
+linkerd-control-plane:
+  identity:
+    issuer:
+      tls:
+        crtPEM: |
+          <contents of the issuer.crt file>
+        keyPEM: |
+          <contents of the issuer.key file>
+  identityTrustAnchorsPEM: |
+    <contents of the ca.crt file>
 ```
 
-- Download the [default values.yaml file](https://github.com/giantswarm/linkerd2-app/blob/master/helm/linkerd2-app/values.yaml) and save it as `my-linkerd-values.yaml` to create a [user configuration](https://docs.giantswarm.io/app-platform/app-configuration/). Review the values in the file. With the default values, linkerd will be installed in High-Availability mode and with CNI plugin enabled.
+- (Optional) Create a file `my-linkerd-values.yaml` with your [user configuration](https://docs.giantswarm.io/app-platform/app-configuration/). Override only the values that are of your interest. With the default values, Linkerd is installed in High-Availability mode and with CNI plugin enabled. Check the full list in the [README](https://github.com/giantswarm/linkerd2-app/blob/master/helm/linkerd2-app/README.md).
```

Comment:
Yes, that's what I wanted to avoid with my change. But your suggestion makes it more clear, thanks!

The only part that I don't fully agree is this one: `Check linked documentation to see how configurations are merged`. I referenced the README of the main chart (remember, this PR moves the upstream chart to a subchart) because values are scoped now under `linkerd-control-plane`. If you read linkerd docs, they won't mention that.

Reply:
You're right, that's a bit ambiguous again. I meant to reference the link in the same paragraph which links to app-platform documentation.

Reply:
Changed it a bit. Does that work?

Reply:
that's great! thank you